### PR TITLE
song-helper

### PR DESCRIPTION
**Pull request by mezgoodle**

### CODE REVIEW

**Comment by mezgoodle** [comment]
So, I think Fakes will be the best practice for testing:
image
In future from Django will be requests to main script, som I created fake Django-server:
```python
from .utils import SpotifyUtils
from .predict import PredictMood


class FakeServer:
    """
    This fake implements the requests from server-side interface.
    This interface is also used by the real implementation
    """
    def __init__(self, spt: SpotifyUtils = None, pt: PredictMood = None) -> None:
        self.spt = spt
        self.pt = pt

    def get_song_with_mood(self, song_id, path) -> dict:
        """
        Django makes request to script to identify song mood
        :param song_id: identifier from Spotify
        :param path: path to dataset
        :return: dictionary with the name of song and mood
        """
        song_with_mood = self.pt.predict_mood(song_id, path)
        return song_with_mood

    def get_song_features(self, song_id) -> dict:
        """
        Django makes request to script to get features of song
        :param song_id: identifier from Spotify
        :return: dictionary with song features
        """
        features = self.spt.get_song_features(song_id)
        return features
```
It was my the first practice with _Fakes_

File: song-helper/fake_server.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class FakeServer:
+    """
+    This fake implements the requests from server-side interface.
+    This interface is also used by the real implementation
+    """
+    def __init__(self, spt: SpotifyUtils = None, pt: PredictMood = None) -> None:
+        self.spt = spt
+        self.pt = pt
+
```
Comment (VsIG-official) [inline song-helper/fake_server.py:13]: Maybe We need more spaces between "def" like in start.py?

Comment (mezgoodle) [reply song-helper/fake_server.py:13]: No, linter shows 0 errors

Comment (VsIG-official) [reply song-helper/fake_server.py:13]: Ok, them

**Review by VsIG-official — APPROVED** [review]

**Review by fhrr-sht — APPROVED** [review]
Max is best!

**Comment by mezgoodle** [comment]
> Max is best!

Thank you)

**Pull request merged**